### DISCUSSION TITLE
Migrate Bitbucket Server and Data Center repos to GitHub.com (public beta)

### DISCUSSION THREAD

**Discussion by tuves** [community/community#60791] [Enterprise]
Today, we're launching public beta support for Bitbucket Server and Bitbucket Data Center migrations in [GitHub Enterprise Importer](https://docs.github.com/en/migrations/using-github-enterprise-importer).

You can now easily use GitHub Enterprise Importer to migrate your source code, revision history, pull requests, reviews and comments when moving to GitHub from your self-hosted Bitbucket instance.

For a step-by-step guide on migrating from Bitbucket Server or Bitbucket Data Center, check out ["Migrating repositories from Bitbucket Server to GitHub Enterprise Cloud"](https://docs.github.com/en/migrations/using-github-enterprise-importer/migrating-repositories-with-github-enterprise-importer/migrating-repositories-from-bitbucket-server-to-github-enterprise-cloud) in the GitHub Docs.

**Comment by Adlinke** [comment]
Therefore last week is not possible migrating Bitbucket service. Thanks for your service.

**Reply by Ademola1987** [reply]
Better

**Comment by kunaguerosantos** [comment]
Was a better idea for peer- peer review. I wasn't expecting that as planned.

**Comment by patrickyibeckwith** [comment]
Hates 92553  Terrorist On My Screen Even Airplane,Telling Me They're Terrorist, Spammed Me Like A Jealous Homo How Can Cops Be Trash Like TP and Delayed single  For Years Is Illegal As... HATE. 24/7 For Nothing.

**Comment by michael-mulder** [comment]
I've been using this tool for a few days now and have 2 suggestions and 1 question.

#### Question

In [step 5](https://docs.github.com/en/migrations/using-github-enterprise-importer/migrating-repositories-with-github-enterprise-importer/migrating-repositories-from-bitbucket-server-to-github-enterprise-cloud#step-5-migrate-a-repository), the last piece of the repository migration is to delete the archive, yet I've ran successful migrations, but the archive still exists on my Bitbucket Server and in S3, what am I missing?

#### Suggestions

1. I noticed that the Bitbucket repository description does not migrate and I'd like to see that get migrated.
2. Is it possible to bring over repository labels and put them as topics on the GitHub repository?

**Reply by Arda01guzel** [reply]
2

**Reply by Arda01guzel** [reply]
5

**Reply by Arda01guzel** [reply]
1

**Reply by Arda01guzel** [reply]
1

**Reply by erniuks12** [reply]
# Ernestas

**Reply by erniuks12** [reply]
# Ernestas

**Reply by boydvan131** [reply]
13

**Reply by timrogers** [reply]
@michael-mulder I'm Tim, and I'm the Product Manager at GitHub for Migration Tools. Thanks for taking the time to join this discussion ❤️ 

To answer your question, you aren't missing anything! That part of the documentation is just wrong and a bit confusing 😓 We delete the archive from your local machine, but you'll need to delete it yourself from blob storage once the migration is finished. We'll update the docs to make this clear in the next couple of days.

As for your suggestions, they make total sense! ❤️ We'll fill those gaps as soon as we can, and will update this thread once that's live. It won't be done in the next week, but we'll get to it soon.

**Reply by timrogers** [reply]
The docs fix is now live ✅

**Reply by Salaheldean** [reply]
رائع

**Reply by Ademola1987** [reply]
Nice

**Reply by Kkdump** [reply]
Logstash configuration

**Reply by timrogers** [reply]
@michael-mulder It turns out that adding descriptions will be harder than we thought, because the archives generated by Bitbucket don't include that data. If you wanted to create a ticket with Bitbucket to raise that, that might help ✨

**Reply by happy560409** [reply]
I am new I have done some thing maybe???

**Reply by andymetri** [reply]
2

**Reply by 64osma** [reply]
Whatsaap hesabımı kullanamıyorum yasaklandı  buradan nasıl kullanabilirim yardımcı olurmusun

**Comment by Arda01guzel** [comment]
1

**Reply by Lk20082** [reply]
I have made two payments on btc pay invoice that I have not been paid and those invoices were paid as per attached proof of bitcoins sent to the wallets that I was prompted to effect payment on.
[6eb5ba61c41b0aa251f43432bf5c41f58428dda8560a6dfed04ecdfbee170967_en_usd.pdf](https://github.com/community/community/files/12136159/6eb5ba61c41b0aa251f43432bf5c41f58428dda8560a6dfed04ecdfbee170967_en_usd.pdf)





(https://github.com/community/community/assets/90218965/f90ad5de-c359-414f-b1a2-325c0f999cda)
![Screenshot_20230720-213301](https://github.com/community/community/assets/90218965/a00e54fb-991f-42c4-a85d-b0c8bf2e86e8)

**Comment by Hobika123** [comment]
Hi friends I’m hobika123

**Comment by yangnicn** [comment]
從我的iPhone傳送Bluechild01 ***@***.***> 於 2023年7月21日 22:58 寫道：﻿
2

**Comment by Kbsq7** [comment]
....[ka]()

**Comment by Kbsq7** [comment]
> ....[ka]()

**Comment by themanofathousandfaces** [comment]
I hope I don't regret it

**Comment by nikhil637** [comment]
Sorry for the mistake

**Comment by Mariamasanenianthio1310** [comment]
Hello, I don't know if this is the right place to ask my question but I'm trying my luck.

I work on a project which consists in inserting my own documents (pdf) in azure openAI and from GPT 3.5 Turbo, I deploy a web application, in which I ask my questions in the assistant and the answers will be based on of the document.

Only I can not savor the distinction between the standard model and the personalized model, because it seems to me they do the same thing.

What I understood is that the custom model if it does not find information in our imported data, it will not be able to look for additional information in the pretrainer model of GTP3 unlike the standard model. Am I wrong or not?

PS: I have already read the documents in the Microsoft site.

**Comment by Spixmacaw** [comment]
responce #2

**Comment by mohdamjadsk** [comment]
> Was a better idea for peer- peer review. I wasn't expecting that as planned.

**Comment by kanoya25** [comment]
ok

**Comment by Angepharel** [comment]
Ok

**Comment by Cyrus361** [comment]
Okay your iPhone 13 pro max issue still active I can unlock it and the process will take 40 minutes  to complete the job

**Comment by reeveng** [comment]
Might as well migrate back, because their homepage is at least less abhorrent :wink:  https://github.com/orgs/community/discussions/categories/feed

**Comment by xieyuheng** [comment]
Please bring back the old feed!

**Comment by Chandrapal-blink** [comment]
@tuves Will it work for migrating repos from bit bucket cloud to github cloud.

**Reply by timrogers** [reply]
No, it won't. At the moment, GitHub Enterprise Importer only supports Bitbucket Server and Bitbucket Data Center - and that functionality is now [generally available](https://github.blog/changelog/2023-09-18-migrate-bitbucket-server-and-data-center-repos-to-github-com/). You can find our recommendations for Bitbucket Cloud [here](https://docs.github.com/en/migrations/overview/migration-paths-to-github#bitbucket-cloud-bitbucketorg-to-githubcom).
